templates/lxc-ubuntu.in

```diff
@@ -92,7 +92,15 @@ configure_ubuntu()
     password=$5
 
     # configure the network using the dhcp
-    cat <<EOF > $rootfs/etc/network/interfaces
+    if [ -d $rootfs/etc/netplan ]; then
```

Comment:
This might be a bit too sensitive.  Maybe `if chroot $rootfs which netplan; then ...` instead?

Reply:
Yeah, sounds good to me.